r/README.md

```diff
@@ -0,0 +1,29 @@
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+# arrow
+
+Low level bindings to the C++ Apache Arrow library
```

Comment:
Maybe "R interface to the ..." or "R integration with...", since the extent of the work involved is a bit more than bindings